plugins/search-backend/src/alpha.ts

```diff
@@ -121,10 +123,16 @@ export default createBackendPlugin({
         const collators = searchIndexRegistry.getCollators();
         const decorators = searchIndexRegistry.getDecorators();
 
-        await searchIndexService.start({
-          searchEngine,
-          collators,
-          decorators,
+        lifecycle.addShutdownHook(() => {
```

Comment:
haha sorry but just for my fragile ocd, can this be below the addStartupHook? 😅

Reply:
Sure :D